src/processor_arm.cpp

```diff
@@ -355,9 +357,11 @@ constexpr auto apple_a13 = armv8_4a_crypto | get_feature_masks(fp16fml, fullfp16
 constexpr auto apple_a14 = armv8_5a_crypto | get_feature_masks(dotprod,fp16fml, fullfp16, sha3);
 constexpr auto apple_a15 = armv8_5a_crypto | get_feature_masks(dotprod,fp16fml, fullfp16, sha3, i8mm, bf16);
 constexpr auto apple_a16 = armv8_5a_crypto | get_feature_masks(dotprod,fp16fml, fullfp16, sha3, i8mm, bf16);
+constexpr auto apple_a17 = armv8_5a_crypto | get_feature_masks(dotprod,fp16fml, fullfp16, sha3, i8mm, bf16); // TODO: may need tweaks?
```

Comment:
Looking at https://github.com/llvm/llvm-project/blob/c66ce080412a7fd73295b7663ab97a81278dd48e/llvm/lib/Target/AArch64/AArch64Processors.td#L394-L443, A16 and A17 have the same features.
```suggestion
constexpr auto apple_a17 = armv8_5a_crypto | get_feature_masks(dotprod,fp16fml, fullfp16, sha3, i8mm, bf16); // TODO: may need tweaks?
```

Reply:
So I guess you meant to suggest to remove the TODO comment?
```suggestion
constexpr auto apple_a17 = armv8_5a_crypto | get_feature_masks(dotprod,fp16fml, fullfp16, sha3, i8mm, bf16);
```

Reply:
No I started to but realized there may be missing features so maybe it should be left in?

Reply:
Ok just noticed you updated it. Maybe it should be an issue instead of a comment in the code.